provisioner/puppet-masterless/provisioner.go

```diff
@@ -73,7 +78,8 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 
 	// Set some defaults
 	if p.config.ExecuteCommand == "" {
-		p.config.ExecuteCommand = "{{.FacterVars}} {{if .Sudo}} sudo -E {{end}}" +
+		p.config.ExecuteCommand = "cd {{.WorkingDir}} && " +
```

Comment:
Hrm yea - I dug through the code and it does not look like there is a way to specify a directory with `RemoteCmd` (which is what gets unwrapped under the hood). Oh well.